src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -54,7 +56,9 @@ export default function withAnnotations<P extends object>(
         handleAnnotationCreate = (eventData: AnnotationActionEvent): void => {
             const { annotation = null, error = null } = eventData;
             const action = this.getAction(eventData);
-            this.setState({ ...this.state, annotation, action, error });
+            const feedId = action === Action.CREATE_START ? uniqueId('annotation_') : this.state.feedId;
```

Comment:
It seems redundant to create another uniqueId since the `annotation` itself has one. Is there a way we could leverage that? Maybe another reason for that is there is a corner case where the API to create is taking a while and in the meantime the user creates another annotation. In this case, the transient feedId be lost

Reply:
Hmm, I think so we can update annotations to send the id on create_start & create_end then update create_end to have the real id?